src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+
+.row {
+  display: flex;
+}
+
+.row__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+  background: url("images/star.svg") no-repeat center;
+  border-radius: 0.5px;
+}
+
+.stars--1 :first-child,
+.stars--2 :nth-child(-n+2),
+.stars--3 :nth-child(-n+3),
+.stars--4 :nth-child(-n+4),
+.stars--5 :nth-child(-n+5) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
```

Comment:
use background property, like in the 13-th line

Reply:
thanks, fixed it